server/.v8flags-1-5.8.283.41.d41d8cd98f00b204e9800998ecf8427e.json

```diff
@@ -0,0 +1 @@
tants_with_cookie","--lazy","--trace_opt","--trace_opt_stats","--trace_file_names","--opt","--always_opt","--always_osr","--prepare_always_opt","--trace_deopt","--trace_stub_failures","--serialize_toplevel","--serialize_eager","--serialize_age_code","--trace_serializer","--max_opt_count","--compilation_cache","--cache_prototype_transitions","--compiler_dispatcher","--compiler_dispatcher_eager_inner","--trace_compiler_dispatcher","--trace_compiler_dispatcher_jobs","--cpu_profiler_sampling_interval","--trace_js_array_abuse","--trace_external_array_abuse","--trace_array_abuse","--trace_debug_json","--enable_liveedit","--trace_side_effect_free_debug_evaluate","--hard_abort","--stack_size","--max_stack_trace_source_length","--always_inline_smi_code","--verify_operand_stack_depth","--clear_exceptions_on_js_entry","--histogram_interval","--trace_object_groups","--heap_profiler_trace_objects","--sampling_heap_profiler_suppress_randomness","--use_idle_notification","--use_ic","--trace_ic","--ic_stats","--native_code_counters","--thin_strings","--trace_weak_arrays","--trace_prototype_users","--use_verbose_printer","--trace_for_in_enumerate","--allow_natives_syntax","--trace_parse","--trace_preparse","--lazy_inner_functions","--aggressive_lazy_inner_functions","--preparser_scope_analysis","--trace_sim","--debug_sim","--check_icache","--stop_sim_at","--sim_stack_alignment","--sim_stack_size","--log_regs_modified","--log_colour","--ignore_asm_unimplemented_break","--trace_sim_messages","--stack_trace_on_illegal","--abort_on_uncaught_exception","--abort_on_stack_overflow","--randomize_hashes","--hash_seed","--random_seed","--trace_rail","--print_all_exceptions","--runtime_call_stats","--runtime_stats","--profile_deserialization","--serialization_statistics","--regexp_optimization","--testing_bool_flag","--testing_maybe_bool_flag","--testing_int_flag","--testing_float_flag","--testing_string_flag","--testing_prng_seed","--startup_src","--startup_blob","--profile_hydrogen_code_stub_compilation","--dump_counters","--dump_counters_nvp","--map_counters","--js_arguments","--log","--log_all","--log_api","--log_code","--log_gc","--log_handles","--log_suspect","--prof","--prof_cpp","--prof_browser_mode","--logfile","--logfile_per_isolate","--ll_prof","--perf_basic_prof","--perf_basic_prof_only_functions","--perf_prof","--perf_prof_unwinding_info","--gc_fake_mmap","--log_internal_timer_events","--log_timer_events","--log_instruction_stats","--log_instruction_file","--log_instruction_period","--redirect_code_traces","--redirect_code_traces_to","--hydrogen_track_positions","--print_opt_source","--trace_elements_transitions","--trace_creation_allocation_sites","--print_code_stubs","--test_secondary_stub_cache","--test_primary_stub_cache","--test_small_max_function_context_stub_size","--print_code","--print_opt_code","--print_opt_code_filter","--print_unopt_code","--print_code_verbose","--print_builtin_code","--sodium","--print_all_code","--predictable","--single_threaded"]
```

Comment:
you could delete this flag since Nodejs just uses it for caching

Reply:
Deleted the flag